tenant-file/portal-app/src/component/add-tenant.tsx

```diff
@@ -0,0 +1,151 @@
+import React from "react";
+
+import ReactDOM from "react-dom";
+
+import { Formik, Field, Form } from "formik";
+
+import { gql, useMutation, useQuery } from '@apollo/client';
+
+const ADD_TENANT = gql`
+    mutation addingATenant($fullName: String!, $phoneNumber: String!, $street: String!, $city: String!, $zip: String!, $bldgId: ID) {
+
+        createTenant(inputTenant: { name: $fullName, phoneNumber: $phoneNumber, currentResidence: { addressInput: { line1: $street, city: $city, state: "TX", postalCode: $zip }, propertyId: $bldgId } } )
```

Comment:
We need to map the apartment number with Line2 of the address type to be in line with the USPS standard